### ISSUE TITLE
g:stardict_prefer_python3 defaults to 1 even if +python3 is not present.

### ISSUE THREAD

**Issue by Raimondi** [phongvcao/vim-stardict#3]
Line 39 in plugin/stardict.vim

```
if !exists('g:stardict_prefer_python3')
    let g:stardict_prefer_python3 = 1
endif
```

 defaults to python3 even if not available, maybe this would be better

```
if !exists('g:stardict_prefer_python3')
    let g:stardict_prefer_python3 = has('python3')
endif
```

**Comment by phongvcao** [comment]
@Raimondi I just quickly changed the code and push up to master. Thanks for the idea.

**Comment by phongvcao** [comment]
This bug was fixed in dcd0049e23acfa4a127b619e40075b66fc5478e4
